RELEASING.md

```diff
@@ -42,28 +42,51 @@ During the week before release, the Release Champion makes changes in preparatio
 - Update [releaseVersions](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/regeneration/release_pipeline_generator.groovy#L6) with correct incoming release versions.
 - Update https://github.com/adoptium/mirror-scripts/blob/master/releasePlan.cfg with expected tag, detail see https://github.com/zdtsw/mirror-scripts/tree/issue/3167#skara-repos-and-processes
 
-Create release branch on below repositories:
+#### Branching message for build related repositories
 
+Post the below  message to the #build & #release channels in Slack:
+
+In Preparation for next weeks release, I'm proposing to branch the
+following repositories, in order that this branch becomes the baseline
+for the release. Shout now if you need to get any PRs merged for the release,
+as today is the last day, the following repositories will be branched:
+  ( temurin-build, ci-jenkins-pipelines, jenkins-helper).
+
+#### Create release branch on below repositories:
+ 
 - temurin-build <https://github.com/adoptium/temurin-build>
 - ci-jenkins-pipelines <https://github.com/adoptium/ci-jenkins-pipelines>
 - jenkins-helper <https://github.com/adoptium/jenkins-helper>
 
-`temurin-build` and `ci-jenkins-pipeline` share the same branch name,`jenkins-helper` might have a different branch name.
-These branches will require at least one approval to push commit to.
+These branches should be named according to the following format (vYYYY.MM.NN) ,e.g v2023.03.01 , whereby the final element is an incremental counter appended to the year and month of the release.
 
-Finally lockdown below repositories
+#### Code Freeze message
 
-- github-release-scripts <https://github.com/adoptium/github-release-scripts>
-- containers <https://github.com/adoptium/containers>
-- installer <https://github.com/adoptium/installer>
-- mirror-script <https://github.com/adoptium/mirror-scripts>
+Paste the below message into the #release channel in Slack:
+
+With under a week to go until releases, we are entering a lockdown period for the following repositories: <a href='https://github.com/adoptium/temurin-build/pulls'>temurin-build</a>, <a href='https://github.com/adoptium/ci-jenkins-pipelines/pulls'>ci-jenkins-pipelines</a>, <a href='https://github.com/adoptium/github-release-scripts/pulls'>github-release-scripts</a>, <a href='https://github.com/adoptium/containers/pulls'>containers</a>, <a href='https://github.com/adoptium/installer/pulls'>installer</a>, <a href='https://github.com/adoptium/mirror-scripts'>mirror-scripts</a>.
+If you need to submit a pr for any of these repos during this period, you should:
+ - Add a comment saying “Approval to merge during the lockdown cycle please” and post in the appropriate slack channel for awareness. This can be done before the PR is finalised
```

Comment:
```suggestion
 - Add a comment saying “Approval to merge during the lockdown cycle please” and post in the appropriate slack channel for awareness. This can be done before the PR is finalised.
```